mp/src/game/shared/momentum/weapon/weapon_mom_concgrenade.cpp

```diff
@@ -122,6 +127,8 @@ void CMomentumConcGrenade::ItemPostFrame()
         ThrowGrenade(GetMaxTimer());
         m_bPrimed = false;
     }
+    else if (!(pPlayer->m_nButtons & IN_ATTACK))
+        m_bRepress = false;
```

Comment:
Always wrap branches in curlies

```suggestion
    else if (!(pPlayer->m_nButtons & IN_ATTACK))
    {
        m_bNeedsRepress = false;
    }
```

The only time you wouldn't need to is if doing a return.